weather.js

```diff
@@ -0,0 +1,70 @@
+const axios = require("axios");
+const dotenv = require('dotenv');
+dotenv.config();
+
+const apiURLPrefix = "http://api.openweathermap.org/";
+
+/**
+ * Runs a conditional to distinguish if Zipcode or address string was entered.
+ * @param {string|number}input
+ * @returns {Promise<void>}
+ */
+async function getWeather(input) {
+    if (isNaN(input)) {
```

Comment:
I added a function here to check for spaces in the formatting. I could go down a rabbit hole of doing additional checks of correct formatting. But I figured this would show that I can get there, and move on. Unless you think it's worth adding a bunch more, or if there is a simpler approach that I am missing?

Reply:
This is good.
Its just something to always think about.